src/parser/name_resolution.c

```diff
@@ -999,6 +1000,12 @@ pt_bind_scope (PARSER_CONTEXT * parser, PT_BIND_NAMES_ARG * bind_arg)
 	  else if (table->node_type == PT_DBLINK_TABLE)
 	    {
 	      assert (spec->info.spec.derived_table_type == PT_DERIVED_DBLINK_TABLE);
+	      if (table->info.dblink_table.is_name)
+		{
+		  pt_resolve_dblink_server_name (parser, table);
```

Comment:
Is it possible that pt_resolve_dblink_server_name() can be failed? If not, assert (pt_has_error (parser)); could be enough